test/api.js

```diff
@@ -478,6 +498,75 @@ describe('/avatars', function() {
   });
 });
 
+describe('/display_name', function() {
+  var tok = token();
+
+  describe('GET', function() {
+    it('should return a displayName', function() {
+      mock.token({
+        user: USERID,
+        email: 'user@example.domain',
+        scope: ['profile:display_name']
+      });
+
+      return db.setDisplayName(USERID, 'Spock')
+      .then(function() {
+        return Server.api.get({
+          url: '/display_name',
+          headers: {
+            authorization: 'Bearer ' + tok
+          }
+        }).then(function(res) {
+          assert.equal(res.statusCode, 200);
+          assert.equal(JSON.parse(res.payload).displayName, 'Spock');
+        });
+      });
+    });
+
```

Comment:
Missing test case is when a display name has not been set.

What should happen? The code currently looks like it will return `{}` if it doesn't exist. Should it instead be a 404? Or stay empty?